api/lib/mno_enterprise/concerns/controllers/deletion_requests_controller.rb

```diff
@@ -68,9 +68,9 @@ def freeze_account
         # Check that the deletion_request has the right status
         if @deletion_request.status == 'pending'
           @deletion_request.freeze_account!
-          format.html { redirect_to @deletion_request, notice: 'Your account has been frozen' }
+          format.html { redirect_to({action: :show, id: @deletion_request.id}, notice: 'Your account has been frozen') }
```

Comment:
No, but only because of the difference of namespace between the controller and the model.  This should be fixed if I put everything on the same namespace.